### DISCUSSION TITLE
Fermata FS Mirror bottom taskbar still showing

### DISCUSSION THREAD

**Discussion by PierreYe** [AndreyPavlenko/Fermata#547] [Q&A]
Hi, I just install Fermata Auto and using the Mirroring function to run 3rd party software. As I'm using it as speedometer for my HUD, I try to hide the Android Auto bottom taskbar using Fermata FS Mirror but it's still showing. I would prefer the apps is display in full screen. I'm using Samsung S21 with Android 13. Thanks for this great software.

**Comment by AndreyPavlenko** [comment]
AFAIK you can't hide the AA task bar without root.
